index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <nav id="navbar" class="nav">
+<ul class="nav-list">
+    <li>
+        <a href="#welcome-section">Welcome</a>
+    </li>
+    <li>
+        <a href="#projects">Projects</a>
+    </li>
+    <li>
+        <a href="#contact">Contact Us</a>
+    </li>
+</ul>
+    </nav>
+    <section id="welcome-section">
+        <h1>This is my Portfolio Page</h1>
+
+    </section>
+    <section id="projects">
+        <p class="project-tile">
+            Lorem ipsum dolor, sit amet consectetur adipisicing elit. Illum obcaecati dolores eveniet repellat magnam omnis enim ab quisquam, nesciunt officiis iure iste voluptas. Modi harum earum corrupti laborum deserunt eum animi nihil labore aliquam!
+        </p>
+        <a href="https://www.freecodecamp.org/learn/responsive-web-design/responsive-web-design-projects/build-a-personal-portfolio-webpage">My Project</a>
+    </section>
```

Comment:
I implore you to please add some styles to this section as well because when you run it on the server, it looks catchy to the user. You can add flexbox, color, background, and align-items properties along with the ones you like to make it a more user-friendly webpage.